ci/pipeline/determine_tests_to_run.py

```diff
@@ -273,6 +273,8 @@ def get_commit_range():
             ):
                 RAY_CI_DOCKER_AFFECTED = 1
                 RAY_CI_LINUX_WHEELS_AFFECTED = 1
+            elif changed_file == ".readthedocs.yaml":
```

Comment:
w00t look like it still builds everything because of ci/ci.sh; if you change the `ci/run` here https://github.com/ray-project/ray/blob/master/ci/pipeline/determine_tests_to_run.py#L339 to just `ci` it should builds only ci and doc

not related to this pr so up to you, if you find yourself continue to iterate, it'll save some ci time